presto-main/src/main/java/com/facebook/presto/cost/StatsNormalizer.java

```diff
@@ -67,7 +72,7 @@ private PlanNodeStatsEstimate normalize(PlanNodeStatsEstimate stats, Optional<Co
             }
 
             SymbolStatsEstimate symbolStats = stats.getSymbolStatistics(symbol);
-            SymbolStatsEstimate normalizedSymbolStats = normalizeSymbolStats(symbol, symbolStats, stats, types);
+            SymbolStatsEstimate normalizedSymbolStats = stats.getOutputRowCount() == 0 ? SymbolStatsEstimate.zero() : normalizeSymbolStats(symbol, symbolStats, stats, types);
```

Comment:
see my other comment: https://github.com/prestodb/presto/pull/11856#discussion_r231049506

I think we should bailout on 0 row count as it suggest CBO misestimation

Reply:
That's what i'm doing. Effectively i'm adding zero symbols stats. I just didn't want to duplicate the loop.